e2e/test/scenarios/onboarding/setup/setup.cy.spec.ts

```diff
@@ -190,12 +191,27 @@ describe("scenarios > setup", () => {
       );
       cy.button("Finish").click();
```

Comment:
in the spirit of not make too many tests, I edited an already existing test that was reaching the end of the flow and added the two extra steps